pkg/util/clusterdata/worker_profiles_task.go

```diff
@@ -69,39 +69,41 @@ func (ef *workerProfilesEnricherTask) FetchData(callbacks chan<- func(), errs ch
 
 	workerProfiles := make([]api.WorkerProfile, len(machinesets.Items))
 	for i, machineset := range machinesets.Items {
+		workerCount := 1
+		if machineset.Spec.Replicas != nil {
+			workerCount = int(*machineset.Spec.Replicas)
+		}
+
+		workerProfiles[i] = api.WorkerProfile{
+			Name:  machineset.Name,
+			Count: workerCount,
+		}
+
 		if machineset.Spec.Template.Spec.ProviderSpec.Value == nil {
+			ef.log.Infof("provider spec is missing in the machine set %q", machineset.Name)
```

Comment:
Now that we ignore the error, I changed it to info. Or should we use warning level here?

Reply:
I think Priority6 (info/print) is good. Currently, we operate only with 3 and 6. The warning is 4. This might confuse the person looking into this.